tests/unit/CoreHeadingTest.php

```diff
@@ -74,6 +90,11 @@ className
 		';
 	}
 
+	/**
+	 * Test the retrieval of core/heading block attributes.
+	 *
```

Comment:
The important part of the doc blocks I was referencing  was the explicit list of "previously untested" attributes, so it's clear at a glance what each test is covering.

Reply:
Apart from line 94, everywhere else 'doc blocks' already contains => explicit list of "previously untested" attributes. 
Added it here as well.